webdriver/tests/bidi/script/call_function/internal_id.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from ... import recursive_compare, any_string
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "return_structure, result_type",
+    [
+        ("[data, data]", "array"),
+        ("new Map([['foo', data],['bar', data]])", "map"),
+        ("({ 'foo': data, 'bar': data })", "object"),
+    ],
+)
+@pytest.mark.parametrize(
+    "expression, type",
+    [
+        ("[1]", "array"),
+        ("new Map([[true, false]])", "map"),
+        ("new Set(['baz'])", "set"),
+        ("{ baz: 'qux' }", "object"),
+    ],
+)
+async def test_remote_values_with_internal_id(
+    call_function, return_structure, result_type, expression, type
+):
+    result = await call_function(
+        f"() => {{ const data = {expression}; return {return_structure}; }}"
+    )
+
+    if result_type == "array":
+        value = [
+            {"type": type, "internalId": any_string},
+            {"type": type, "internalId": any_string},
+        ]
+    else:
+        value = [
+            ["foo", {"type": type, "internalId": any_string}],
+            ["bar", {"type": type, "internalId": any_string}],
+        ]
+
+    recursive_compare({"type": result_type, "value": value}, result)
```

Comment:
Do we really need to check the return type here? It looks like comparing the value will enough. Can simplify the test, and check only the internal IDs, without data types?

Reply:
True, the `result_type` is not really important here, I've removed it, but I think it is still good to check that the `type` is present in the `value` with `internalId`

Reply:
It can be removed from the parameters as well them